config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -158,6 +158,7 @@ config CHIP_DFU_OVER_BT_SMP
 	# Enable custom SMP request to erase settings partition.
 	select MCUMGR_GRP_ZBASIC
 	select MCUMGR_GRP_ZBASIC_STORAGE_ERASE
+	select MCUMGR_TRANSPORT_BT_REASSEMBLY
```

Comment:
Could we change this from select to imply? I can see there might be some advantages from using it, but I would not say this is crucial for SMP to work and we have to force it without allowance to disable it.